views/partials/code-mirror-deps.ejs

```diff
@@ -0,0 +1,24 @@
+<!-- editor -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.39.0/codemirror.min.js"></script>
```

Comment:
All externally linked libraries should be stored locally in the app. There is no advantage in using cloud versions in a local app like dynamodb-admin and also app should not require online access.

Also, ace editor files should be removed if no longer used.